@kiva/kv-components/vue/KvSelect.vue

```diff
@@ -6,6 +6,7 @@
 				:id="id"
 				:disabled="disabled"
 				:value="value"
+				role="select"
```

Comment:
This role can be removed as it is redundant on a `<select>` element

Reply:
Nice point, thanks. Is fixed.